src/lib.rs

```diff
@@ -69,15 +69,22 @@ pub struct DebuggerLogger {
 /// info!("Hello, world!");
 /// debug!("Hello, world, in detail!");
 /// ```
-pub static DEBUGGER_LOGGER: DebuggerLogger = DebuggerLogger { _priv: () };
+pub static DEBUGGER_LOGGER: DebuggerLogger = DebuggerLogger { force_log_without_debugger: std::sync::atomic::AtomicBool::new(false) };
+
+impl DebuggerLogger {
+    /// Set logging wihout debugger attached flag.
+    pub fn set_force_log_without_debugger(&self,v:bool) {
+        self.force_log_without_debugger.store(v, std::sync::atomic::Ordering::Relaxed)
+    }
+}
 
 impl log::Log for DebuggerLogger {
-    fn enabled(&self, metadata: &Metadata) -> bool {
-        metadata.level() <= Level::Debug
+    fn enabled(&self, _metadata: &Metadata) -> bool {
+        true
```

Comment:
Should this be `metadata.level() <= Level::Debug && self.force_log_without_debugger.load(...)`?

Reply:
Because logging level checked in crate log here we just should ensure own condition. Will fix.